docs/repositories-configure/managing-branches.md

```diff
@@ -16,7 +13,7 @@ To change the default branch of your repository or enable analysis on other bran
 
     Enabling a new branch triggers an initial analysis of that branch and the analysis results and information for that branch will become available after the analysis is complete.
 
-1.  To change the default branch on Codacy, click the corresponding **Make default** link that appears when you hover the column **Default**.
+1.  To change the default branch on Codacy, click the corresponding **Make default** link that appears when you hover the column **Default**. Changing the default branch on Codacy doesn't change the default branch on your Git provider.
```

Comment:
While we're clarifying what a default branch is, I realized that we should probably mention this. It's not obvious

Reply:
Yes, good idea! :ok_hand: